atest/test/02_Content_Keywords/virtual_mouse.robot

```diff
@@ -53,16 +54,73 @@ Drag And Drop With Strict
 
 Drag and Drop with coordinates
     ${obj_center} =    Get Boundingbox    id=draggable    ALL    evaluate    ${Center_Func}
+    ${obj_dim} =    Get Boundingbox    id=draggable    ALL    evaluate    ${Dim_Func}
     ${dest_center} =    Get Boundingbox    id=clickWithOptions    ALL    evaluate    ${Center_Func}
+    # Tests with implicit argument drop=True
     Drag And Drop By Coordinates
     ...    from_x=${obj_center}[x]    from_y=${obj_center}[y]
     ...    to_x=${dest_center}[x]    to_y=${dest_center}[y]    steps=200
-    ${obj_center} =    Get Boundingbox    id=draggable    ALL    evaluate    ${Center_Func}
-    ${x_diff} =    Evaluate    ${obj_center}[x] - ${dest_center}[x]
-    ${y_diff} =    Evaluate    ${obj_center}[y] - ${dest_center}[y]
-    Log    X-Diff: ${x_diff}, Y-Diff: ${y_diff}
-    Should Be True    ${x_diff} < 1 or ${x_diff} > -1
-    Should Be True    ${y_diff} < 1 or ${y_diff} > -1
+    Assert Position    ${dest_center}[x]    ${dest_center}[y]    ${tol}
+    Drag And Drop By Coordinates
+    ...    from_x=${dest_center}[x]    from_y=${dest_center}[y]
+    ...    to_x=${obj_center}[x]    to_y=${obj_center}[y]    steps=200
+    Assert Position    ${obj_center}[x]    ${obj_center}[y]    ${tol}
+    # Tests with explicit values True or False for argument drop
+    # "Start coordinates" of draggable object:
+    ${x1} =    Set Variable    ${obj_center}[x]
+    ${y1} =    Set Variable    ${obj_center}[y]
+    ${width} =    Set Variable    ${obj_dim}[width]
+    ${height} =    Set Variable    ${obj_dim}[height]
+    Log    Draggable object: ${obj_center}
+    # coordinates where to drag in relative values:
+    ${x2} =    Evaluate    ${x1} + 0.1 * ${width}
+    ${y2} =    Evaluate    ${y1} - 1.0 * ${height}
+    ${x3} =    Evaluate    ${x2} + 0.06 * ${width}
+    ${y3} =    Evaluate    ${y2} + 1.2 * ${height}
+    ${x4} =    Evaluate    ${x3} + 0.08 * ${width}
+    ${y4} =    Set Variable    ${y3}
+    ${x5} =    Set Variable    ${x4}
+    ${y5} =    Set Variable    ${height}
+    ${steps} =    Set Variable    200
+    ${time} =    Set Variable    1
+
+    # Temporary deactivated. The very next call of "Drag And Drop By Coordinates" would make MacOS fail.
```

Comment:
These should not be commented out. Do we still need these?

Reply:
We can divide it in an own issue. 
There is a bug when ones repeats the call of the keyword certain times.
Mac OS is the most sensitive OS in that case but the failures also occur with WIN and Ubuntu some calls later.
I'll do some more tests anyway when I use the new keyword argument with my application.
Maybe that I hereby get more insights when the issue occurs or not.
So I'd delete the commented part for now for that we have clean code.

Reply:
Need to be patient with this PR :-)
Build processes and test install were successful but there was one failure dealing with uploading artifacts,
see test-install (windows-latest, 3.9, 16.x) 

For more detailed logs during the artifact upload process, enable step-debugging: https://docs.github.com/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging#enabling-step-debug-logging

![grafik](https://user-images.githubusercontent.com/89480399/155903249-071e8fb6-4710-433c-bc43-687b6ad1f060.png)

If I should take further steps please let me know.

Reply:
1) Tests
Generally I feel that commenting code/test out is not good idea. Either code/tests are valid and is run or it is not needed and should be deleted. It can be always found from the history if there is need to dig it out. Many cases there is not
If these are valid test, then we should run them. We have test which are not successful in some platforms and those test can be skipped by using `no-windows-support` or `no-mac-support` tags. 
I think we should enable the tesst, but skip them in Mac and Win OS.

2) Failures
GitHub actions have had some regression in performance and some steps in CI just fail without a good reason. Currently actions seems to be more stable, but if you see error in steps which is running tests, we can ignore the error.